use/extension.go

```diff
@@ -96,5 +98,16 @@ func (s *Statement) Origin() (origin *Origin, err error) {
 
 // Extension func for parser
 func (c *Context) Extension(file *dockerfile.Dockerfile, line int, tokens *dockerfile.Tokens) dockerfile.Statement {
-	return &Statement{Context: c, Line: line, Reference: tokens.NextLine()}
+	instruction := tokens.NextLine()
+	var reference string
+	dropEntrypoint := false
+	pos := strings.LastIndex(instruction, "|")
+	if -1 == pos {
+		reference = strings.TrimSpace(instruction)
+	} else {
```

Comment:
Thanks for the hint with `Trim` here - applied to https://github.com/tueftler/doget/releases/tag/v0.6.0